src/get-started/index.njk

```diff
@@ -0,0 +1,237 @@
+</p>
+
+<h3 class="govuk-heading-m" id="Low-fidelity-prototyping-tools">
+Low-fidelity prototyping tools
+</h3>
+
+<h3 class="govuk-heading-s" id="sketches">
+Sketches
+</h3>
+
+<p class="govuk-body">
+Sketched prototypes, using pen and paper, are the easiest way to start exploring design ideas.
+</p>
+
+<p class="govuk-body">
+Sketches are cheap and easy to produce and can be discarded if they don’t meet your needs. This makes them perfect for the early stages of designing a service or product.
+</p>
+
+<h3 class="govuk-heading-s" id="google-suite">
+Google Suite
+</h3>
+
+<p class="govuk-body">
+Designers with access to Google Suite have a range of tools available for creating journey maps, flow diagrams, quick sketches, and forms. 
+</p>
+
+<h3 class="govuk-heading-s" id="mural">
+Mural
+</h3>
+
+<p class="govuk-body">
+Mural is available for use on HMRC devices and we recommend using it for creating flow diagrams, journey maps, sketching out design ideas and creating simple screens. 
+</p>
+
+<p class="govuk-body">
+Full licences may not be available for every designer, please speak to the design leadership in your programme or P&UCD Ops to find out what is available for you.
+</p>
+
+<p class="govuk-body">
+  Mural templates for low-fidelity prototyping:
+  
+  <ul class="govuk-list govuk-list--bullet">
+    <li><a href="https://github.com/dashouse/govuk-design-system-flow-diagrams" class="govuk-link">Flow diagrams for GOV.UK services</a></li>
+    <li><a href="https://github.com/clare-brown/govuk-designsystem-flow-diagram-mural" class="govuk-link">GOV.UK elements in Mural</a></li>
+    </ul>
+    </p>
+
+<h2 class="govuk-heading-l" id="High-fidelity-prototypes">
+ High-fidelity prototypes
+ </h2>
+ 
+ <p class="govuk-body">
+ High-fidelity prototypes are detailed, coded prototypes that mimic the behaviour of ‘finished’ services. High-fidelity prototypes will help you to perform user testing and demonstrate the service to stakeholders.
+ </p>
+ 
+ <p class="govuk-body">
+ HMRC’s preferred approach is to use the GOV.UK Prototype Kit. You will need a device, such as a MacBook, that isn’t managed by HMRC so you can install the necessary software. 
+ </p>
+ 
+ <p class="govuk-body">
+ You can <a href="https://design-system.service.gov.uk/get-started/prototyping/">follow instructions</a> in the GOV.UK Design System to get the kit running on your unmanaged device.
+ </p>
+ 
+ <h3 class="govuk-heading-m" id="Prototype-support">
+ Prototype support
+ </h3>
+ 
+ <p class="govuk-body">
+ The GOV.UK Prototype Kit is used across governments and there is a significant amount of community support available if you get stuck or need help. 
+ </p>
+ 
+ <p class="govuk-body">
+ Inside HMRC you can reach out in our #community-prototype Slack channel or speak to members of HMRC’s Design Resources Team. Support is also available on the cross-government Slack in the #govuk-prototype-kit channel.
```

Comment:
We've linked to Slack channels above. Should we be doing the same for these ones?